src/action/linux/start_systemd_unit.rs

```diff
@@ -17,11 +17,27 @@ pub struct StartSystemdUnit {
 impl StartSystemdUnit {
     #[tracing::instrument(level = "debug", skip_all)]
     pub async fn plan(unit: impl AsRef<str>) -> Result<StatefulAction<Self>, ActionError> {
+        let unit = unit.as_ref();
+        // If `systemctl is-active $UNIT` has exit status 3, it doesn't exist
+        let output = Command::new("systemctl")
+            .arg("is-active")
+            .arg(unit)
+            .output()
+            .await
+            .map_err(ActionError::Command)?;
+
+        let state = if output.status.success() {
+            tracing::debug!("Starting systemd unit `{}` already complete", unit,);
```

Comment:
Nit:

```suggestion
            tracing::debug!("Starting systemd unit `{}` already complete", unit);
```

Reply:
How does Rustfmt not get this...

Reply:
Macros are hard for rustfmt, unfortunately.